bin/radical-pilot-command

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+__copyright__ = "Copyright 2024, http://radical.rutgers.edu"
+__license__   = "MIT"
+
+
+import sys
+import time
+
+import radical.utils   as ru
+import radical.pilot   as rp
```

Comment:
not used in the code

Reply:
or maybe keep it, and then use it for channel name: instead of `'control_pubsub'` use `rp.CONTROL_PUBSUB`?

Reply:
Thanks, I did so (use the constant)